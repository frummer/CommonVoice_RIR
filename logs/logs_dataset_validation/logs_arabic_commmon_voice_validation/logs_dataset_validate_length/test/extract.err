Processing subdirectories:   0%|          | 0/6 [00:00<?, ?dir/s]Processing subdirectories:  17%|█▋        | 1/6 [00:00<00:03,  1.37dir/s]Processing subdirectories:  33%|███▎      | 2/6 [00:01<00:02,  1.36dir/s]Processing subdirectories:  50%|█████     | 3/6 [00:03<00:03,  1.22s/dir]Processing subdirectories:  67%|██████▋   | 4/6 [00:04<00:02,  1.11s/dir]Processing subdirectories:  83%|████████▎ | 5/6 [00:05<00:01,  1.10s/dir]Processing subdirectories: 100%|██████████| 6/6 [00:06<00:00,  1.07s/dir]Processing subdirectories: 100%|██████████| 6/6 [00:06<00:00,  1.05s/dir]
