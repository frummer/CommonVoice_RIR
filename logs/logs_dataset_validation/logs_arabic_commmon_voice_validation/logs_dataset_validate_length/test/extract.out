Validating Datasets subdirs length
Script started at: Wed Feb  5 12:09:18 PM EST 2025
Base Directory: /export/fs05/afrumme1/babylon_datasets_1/arabic/test_01_02_2025_22_25_47_8_4_20_10_15_10/test
source2_reverb: 5216 .wav files
source2: 5216 .wav files
source1: 5216 .wav files
compressed_mixture: 5216 .wav files
mixture: 5216 .wav files
source1_reverb: 5216 .wav files
